import-areas/import-areas.go

```diff
@@ -338,7 +339,7 @@ func main() {
 
 	logData := log.Data{"AreaData": englandData}
 
-	if err = session.DB("areas").C("areas").Insert(englandData); err != nil {
+	if _, err = session.DB("areas").C("areas").Upsert(bson.M{"id": "E92000001"}, englandData); err != nil {
```

Comment:
Are these ID's constant (or are the likely to change)? Better off having these in a map[string]string like:

`var regionalIDs = map[string]string{ "England": "E92000001", "Wales": "W92000004" }`

And referencing that map rather than inlining.